File: example/src/main/java/com/xwray/groupie/example/InsetItemDecoration.java
package com.xwray.groupie.example;

import android.support.annotation.ColorInt;
import android.support.annotation.Dimension;

public class InsetItemDecoration extends com.xwray.groupie.example.core.decoration.InsetItemDecoration {
    public InsetItemDecoration(@ColorInt int backgroundColor, @Dimension int padding) {
        super(backgroundColor, padding, MainActivity.INSET_TYPE_KEY, MainActivity.INSET);
    }
}

File: example/src/main/java/com/xwray/groupie/example/InsetItemDecoration.kt
package com.xwray.groupie.example

import androidx.annotation.ColorInt
import androidx.annotation.Dimension

class InsetItemDecoration(@ColorInt backgroundColor: Int, @Dimension padding: Int) :
        com.xwray.groupie.example.core.decoration.InsetItemDecoration(backgroundColor, padding, INSET_TYPE_KEY, INSET)
